### DISCUSSION TITLE
Example of using GreasedLine with react-babylonjs

### DISCUSSION THREAD

**Discussion by don-sitebionics** [brianzinn/react-babylonjs#314] [Q&A]
I am able to get a GreasedLine to work with procedural BabylonJS but having trouble getting this to work declaratively with react.  My best guess at it is...

<greasedLineMesh name="myGreasedLine" _options={{ points: [new Vector3(0, 0, 0), new Vector3(10, 10, 10), new Vector3(0, 0, 1)]}} />

...but this doesn't show up.

Anyone ever get GreasedLine to work with react?

**Comment by brianzinn** [comment]
I never did actually try it out.  It was just added to "API", since it was generated.  What you have looks like it ought to work.  Looking deeper into the babylon.js code - I see that I should also be supporting the builder likely (currently I only support MeshBuilder).  I looked at that builder and it is simply doing a `new GreasedLineMesh(...)`, so I would expect that to work.  the "points" can be `Vector3[]` or `number[]`, `number[][]`, etc.  I'll spend some time here at some point - i'm looking today at adding WebGPUEngine support.. finally.

**Reply by don-sitebionics** [reply]
I'll try to dig in and debug this some more but it will probably be later in the week.
